controllers/contacts.js

```diff
@@ -96,7 +95,7 @@ module.exports = function(app, config) {
 						query+= " `organization`.`organization_name` as `contact_orgranization`,";
                                                 query+= " concat(`cnt2`.firstname,' ',`cnt2`.lastname) as `contact_report_to`,";
                                                 query+= " case when (`user`.`user_name` not like '')";
-						query+= " then";
+                                                query+= " then";
 						query+= " `user`.`user_name`";
```

Comment:
Looks like still having the indentation issue. 
Look at the line 
```
query+= " concat(`cnt2`.firstname,' ',
```
and
```
query+= " case when (`user`.`user_name` not like '')";
```

and the line 
```
query+= " then";
```

they all are having an extra space at the beginning.